src/lib.rs

```diff
@@ -159,6 +159,13 @@ pub struct Package {
     pub name: String,
     /// Version given in the `Cargo.toml`
     pub version: String,
+    /// The [SPDX license expression][spdx-expressions] for this package.
+    ///
+    /// [spdx-expressions]: https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60
```

Comment:
I have no idea about this topic. But doesn't that mean that we might in the future expose a more structured way to access the license (so not just a `String`)?

If so, please create a `License` type that contains a private `String` field and exposes that via a `Display` impl.